client/src/components/whiteboard.js

```diff
@@ -17,7 +17,7 @@ class Whiteboard extends React.Component {
 
   render() {
     return (
-      <div>
+      <div id="canvasWrapper">
```

Comment:
Let's prefer classes over IDs. You can't have more than one of the same ID on a page. What if we wanted two whiteboards at the same time?